### PR TITLE
docs: add basics to readme and contributing guide

### PR DESCRIPTION
**Pull request by sethmerrick**
This change is [Reviewable](https://reviewable.io/reviews/storyshop/api/44)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+MIT
+
+## App Repo
+
+Though it is not a requirement in order to install & run the code in this backend `api` repo, there is a frontent `app` at [https://github.com/StoryShop/app](https://github.com/StoryShop/app), if one wants to spin up the _complete_ application.
+
+## Installation
+
+First you need to update your `/etc/hosts` file to include `dev.storyshopapp.com`, like so:
```
Comment (joshdmiller) [inline README.md]: This step is not necessary for the backend.

Comment (joshdmiller) [inline README.md]: This step is not necessary for the backend.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+$ npm install
+$ npm run data
+```
+
+In order to have the api become fully-functional and properly connected to our cloud services, there are a number of environmental variables kept outside of the public repo in a file at `/run.sh`. Please contact an org admin to obtain the contents of this file.
+
+With the above completed, the final step to run the back end is to execute the `/run.sh` script like so:
+
+```bash
+$ bash run.sh
```
Comment (joshdmiller) [inline README.md:33]: Or `sh run.sh` or `./run.sh`.

**Review by joshdmiller — COMMENTED** [review]
I put in a couple minor comments. There also appears to be the addition of an empty file. And we might want to link to the git contribution guide here too.

**Comment by sethmerrick** [comment]
I included the empty file as a placeholder because it's linked to from the readme. Do you advise I omit the empty file _and_ the mention in the readme until they become relevant, or should I add "Coming Soon" markdown to the empty file?

**Comment by joshdmiller** [comment]
What will eventually be in the file?

**Comment by sethmerrick** [comment]
Ah, this was actually a mistake. Thanks! That file should include the top bit of https://github.com/StoryShop/app/blob/documentation/docs/getting_started_guide.md, from #Contributing to before ##Introduction - including the link to the guide. If we do not intend to add to that file moving forward, I can just save a click and move that brief copy to readme.

**Pull request merged**